src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+              var o = t.match(/((?:\-|\+)?\d*\.?\d*)(.*)/),
+                r = +o[1],
+                s = o[2];
+              if (!r) return t;
+              if (0 === s.indexOf("%")) {
+                var a = void 0;
+                switch (s) {
+                  case "%p":
+                    a = n;
+                    break;
+                  case "%":
+                  case "%r":
+                  default:
+                    a = i;
+                }
+                return (K(a)[e] / 100) * r;
+              }
+              if ("vh" === s || "vw" === s) {
+                return (
+                  (("vh" === s
+                    ? Math.max(
+                        document.documentElement.clientHeight,
+                        window.innerHeight || 0
+                      )
+                    : Math.max(
+                        document.documentElement.clientWidth,
+                        window.innerWidth || 0
+                      )) /
+                    100) *
+                  r
+                );
+              }
+              return r;
+            })(t, o, e, n);
+          });
+      })).forEach(function (t, e) {
+        t.forEach(function (n, i) {
+          _t(n) && (o[e] += n * ("-" === t[i - 1] ? -1 : 1));
+        });
+      }),
+      o
+    );
+  }
+  var Ot = {
+      placement: "bottom",
+      positionFixed: !1,
+      eventsEnabled: !0,
+      removeOnDestroy: !1,
+      onCreate: function () {},
+      onUpdate: function () {},
+      modifiers: {
+        shift: {
+          order: 100,
+          enabled: !0,
+          fn: function (t) {
+            var e = t.placement,
+              n = e.split("-")[0],
+              i = e.split("-")[1];
+            if (i) {
+              var o = t.offsets,
+                r = o.reference,
+                s = o.popper,
+                a = -1 !== ["bottom", "top"].indexOf(n),
+                l = a ? "left" : "top",
+                c = a ? "width" : "height",
+                u = {
+                  start: z({}, l, r[l]),
+                  end: z({}, l, r[l] + r[c] - s[c]),
+                };
+              t.offsets.popper = X({}, s, u[i]);
+            }
+            return t;
+          },
+        },
+        offset: {
+          order: 200,
+          enabled: !0,
+          fn: function (t, e) {
+            var n = e.offset,
+              i = t.placement,
+              o = t.offsets,
+              r = o.popper,
+              s = o.reference,
+              a = i.split("-")[0],
+              l = void 0;
+            return (
+              (l = _t(+n) ? [+n, 0] : Nt(n, r, s, a)),
+              "left" === a
+                ? ((r.top += l[0]), (r.left -= l[1]))
+                : "right" === a
+                ? ((r.top += l[0]), (r.left += l[1]))
+                : "top" === a
+                ? ((r.left += l[0]), (r.top -= l[1]))
+                : "bottom" === a && ((r.left += l[0]), (r.top += l[1])),
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)